README.md

```diff
@@ -96,4 +111,4 @@ npm install
 PORT=3001 npm run start
 ```
 
-You can now visit http://localhost:3001 to try out the dev wallet.
+You can now visit http://localhost:3001 to try out the dev-wallet.
```

Comment:
They no longer need to clone the `orodio` repo -- they can just navigate to http://localhost:8701/harness.